web/src/services/i18n/translations/ja.yml

```diff
@@ -124,8 +124,23 @@ Choose layer to add: 追加するレイヤーを選択
 layer name: レイヤー名
 Layer name: レイヤー名
 Inspector: インスペクター
+Data source: ''
```

Comment:
I think we should keep japanese translataions provided always. Could you fill these with any translation tools and ask @asahina820 help review?